app/models/ext_management_system.rb

```diff
@@ -27,6 +27,9 @@ def self.supported_types_and_descriptions_hash
 
   belongs_to :provider
   belongs_to :tenant
+  include ComplianceMixin
+  include CustomAttributeMixin
+  after_initialize :load_custom_attributes
```

Comment:
@alongoldboim you probably want to move this functionality down to the container manager.  If we need to expose this type of relationship for all managers, we can look at that later.

Also, `miq_report`, another class that uses `load_custom_attributes_for` via the `CustomAttributeMixin`, uses lazy loading to pull in the custom attributes only when they're accessed.

If there's definitely not a way to do this lazily, then at least having it isolated in the container manager would be better than exposing in `ext_management_system`.